src/databases/Blueprint/avtBlueprintTreeCache.C

```diff
@@ -1031,13 +1031,82 @@ avtBlueprintTreeCache::SetProtocol(const std::string &protocol)
     m_protocol = protocol;
 }
 
+//-----------------------------------------------------------------------------
+// This uses a mapping scheme created by ascent + conduit
+// Note: We will support explicit maps from the bp index in the future.
+// TODO: We should cache this, not regen every fetch
+//-----------------------------------------------------------------------------
+//-----------------------------------------------------------------------------
+void gen_domain_to_file_map(int num_domains,
```

Comment:
If external linkage isn't needed would suggest making this `static` here. I think that is a good practice whenever it makes sense to do so though I am not honestly sure whether global compiler flags in `fvisibility=hidden` already basically achieve this.

Reply:
will do, thanks for the suggestion!